README.md

```diff
@@ -55,7 +55,33 @@ Run `xattr -d com.apple.quarantine path-to-whisky`
 
 There is an issue where the libraries for font rendering are not properly bundled yet. You will need to install freetype.
 
-- Open an x86 terminal session with `arch -x86_64 zsh`
-- Install brew `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"` (If you run into issues, let me know)
+
+### Setup your development and Homebrew environment
```

Comment:
These are steps that need to be followed in order. So convert them from unordered bullets to ordered numerals.